sys/debian.sh

```diff
@@ -31,7 +31,9 @@ for a in ${PKGDIR}/usr/lib/libr*.so.* ; do
 done
 
 echo "[debian] building radare2 package..."
+./configure
```

Comment:
isn't one `configure` enough? Also, I think it should be moved before the `make install` just few lines above.

Reply:
yeah, but it doesnt hurts, its cleaner to read imho

Reply:
nvm ive seen the issue :D